src/mesh/tests/RTreeTests.cpp

```diff
@@ -9,26 +11,169 @@ namespace bgi = boost::geometry::index;
 
 BOOST_AUTO_TEST_SUITE(MeshTests)
 BOOST_AUTO_TEST_SUITE(RTree)
+BOOST_AUTO_TEST_SUITE(BGAdapters)
+
+BOOST_AUTO_TEST_CASE(VectorAdapter)
```

Comment:
What is this Test for?

Reply:
This is a test for the vertex adaptation to boost geometry.
@floli wrote this test and the one for the eigen vector. I grouped all of the adapter tests into a suite, therefor the diff.